Tasks/Xcode/Strings/resources.resjson/es-es/resources.resjson

```diff
@@ -45,8 +45,8 @@
   "loc.input.help.provisioningProfileUuid": "(Opcional) Especifique el UUID de un archivo de aprovisionamiento instalado que se usará para la compilación. Use tareas de compilación independientes con diferentes esquemas o destinos a fin de especificar perfiles de aprovisionamiento independientes por destino en una sola área de trabajo (iOS, tvOS, watchOS).",
   "loc.input.label.teamId": "Id. de equipo",
   "loc.input.help.teamId": "(Opcional, a menos que sea miembro de varios equipos de desarrollo). Especifique el identificador de equipo de desarrollo de 10 caracteres.",
-  "loc.input.label.destinationPlatformOption": "Plataforma de destino",
-  "loc.input.help.destinationPlatformOption": "Seleccione la plataforma del dispositivo de destino que se va a usar para las pruebas de IU cuando el dispositivo de compilación genérico no es válido. Elija \"Personalizar\" para especificar una plataforma que no está incluida en la lista. Si se selecciona \"Predeterminado\", no se establecerán simuladores ni dispositivos como destino.",
+  "loc.input.label.destinationPlatform": "Plataforma de destino",
+  "loc.input.help.destinationPlatform": "Seleccione la plataforma del dispositivo de destino que se va a usar para las pruebas de IU cuando el dispositivo de compilación genérico no es válido. Elija \"Personalizar\" para especificar una plataforma que no está incluida en la lista. Si se selecciona \"Predeterminado\", no se establecerán simuladores ni dispositivos como destino.",
   "loc.input.label.destinationPlatform": "Plataforma de destino personalizada",
```

Comment:
Looks like in a lot of these resource files you didn't remove the old destinationPlatform help and label.

Reply:
Good catch!